README.md

```diff
@@ -3,18 +3,146 @@
   <p> <img src="https://media.giphy.com/media/hvRJCLFzcasrR4ia7z/giphy.gif" alt="Waving Hand" width="25px" height="25px"> Welcome to Reactjs Developer Community Kenya Website!</p>
 </h1>
 
-This is the repo for the showcase website of the React Developer Community in Kenya
+This is the repository for the showcase website of the React Developer Community in Kenya.
 
 Check out the live version [HERE](https://reactdevske.vercel.app/)
 
-## Contributing to the project
+---
+
+## Table of Contents
+
+- [Project Overview](#project-overview)
+- [Features](#features)
+- [Getting Started](#getting-started)
+- [Contributing to the Project](#contributing-to-the-project)
+- [Technologies Used](#technologies-used)
+- [Figma Design File](#figma-design-file)
+- [FAQs](#faqs)
+- [License](#license)
+- [Contact](#contact)
+
+---
+
+## Project Overview
+
+The React Developers Kenya Website is a platform designed to showcase the vibrant React.js developer community in Kenya. It serves as a hub for developers to connect, share knowledge, and collaborate on projects. The website is open-source and welcomes contributions from developers worldwide and across the country.
+
+---
+
+## Features
+
+- Showcase of the React Developer Community in Kenya.
+- Open-source and community-driven.
+- Responsive design for seamless use across devices.
+- Easy-to-navigate structure for contributors and users.
+- Integration with GitHub for issue tracking and contributions.
+
+---
+
+## Getting Started
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+Ensure you have the following installed:
+
+- [Node.js](https://nodejs.org/)
+- [Git](https://git-scm.com/)
+
+### Installation
+
+1. Clone the repository:
+
+```bash
+   git clone https://github.com/reactdeveloperske/reactdevske-website.git
+```
+
+2. Navigate to the project directory:
+
+```bash
+   cd reactdevske-website
+```
+
+3. Install dependencies:
+
+```bash
+   npm install
+```
+
+4. Start the development server:
+
+```bash
+   npm run dev
+```
+
+5. Open your browser and navigate to `http://localhost:3000`.
+
+---
+
+
+---
+
+## Contributing to the Project
 
 Contributions are always welcome!
 
-See [`contributing.md`](https://github.com/reactdeveloperske/reactdevske-website/blob/main/CONTRIBUTING.md) for ways to get started.
+### How to Contribute
```

Comment:
@festus-sulumeti 

For the contribution guidelines section, emphasize on the detailed instructions that are available on the contribution guidelines. Some steps are missing from this summary which are very important and are addressed on the contribution guidelines,